include/mbedtls/ctr_drbg.h

```diff
@@ -175,7 +189,11 @@ typedef struct mbedtls_ctr_drbg_context {
                                                   * This is the maximum number of requests
                                                   * that can be made between reseedings. */
 
+#if defined(MBEDTLS_AES_C)
     mbedtls_aes_context MBEDTLS_PRIVATE(aes_ctx);        /*!< The AES context. */
+#else
+    mbedtls_ctr_drbg_psa_context MBEDTLS_PRIVATE(psa_ctx); /*!< The PSA context. */
```

Comment:
In my initial design there was no such structure indeed. I added it when I added the init/free functions:
- ctr_drbg_setup_psa_context()
- ctr_drbg_destroy_psa_contex()

At that point I thought it looked nicer/clearer to pass a single structure to those functions. Of course I could have used either arguments or the container `struct mbedtls_ctr_drbg_context` (but not in `block_cipher_df()`). Perhaps in the end I was simply "inspired" by the legacy AES context.

Well, it turns out that this is clearly not a hardly thought design choice. So if for any reason you think it's better to keep standalone values instead of that structure just let me know ;)

Reply:
Ah, actually when I saw the setup/destroy function I thought "ah that's why" and meant to go back and remove this question, but evidently didn't. Fine with me!